app/pages/angelfish.tsx

```diff
@@ -25,61 +25,66 @@ const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
   const [region, selectedRegion] = useState<DhbRegionId>("auckland");
 
   return (
-    <div className="page">
+    <main className="page flex align-items-center">
```

Comment:
Manually styling this as flex for now, instead of assuming it in the page class name